compose/case-assignment.yml

```diff
@@ -0,0 +1,35 @@
+---
+version: '2.4'
+services:
+  manage-case-assignment:
+    image: hmctspublic.azurecr.io/aac/manage-case-assignment:latest
+    ports:
+      - 4454:4454
+      - 5005:5005
+    environment:
+      IDAM_OIDC_URL: http://idam-api:5000
+      IDAM_API_URL: http://idam-api:5000
+      IDAM_CLIENT_ID: aac_manage_case_assignment
+      IDAM_CLIENT_SECRET: "${IDAM_KEY_AAC_MANAGE_CASE_ASSIGNMENT}"
+      IDAM_CLIENT_REDIRECT_URL: http://localhost:3333/oauth2/callback
+      IDAM_CAA_USERNAME: "caa-caseworker@mailnesia.com"
+      IDAM_CAA_PASSWORD: "Password12"
+      IDAM_NOC_APPROVER_USERNAME: "approver-noc@mailnesia.com"
+      IDAM_NOC_APPROVER_PASSWORD: "Password12"
+      IDAM_SYSTEM_USER_USERNAME: "caa-caseworker@mailnesia.com"
+      IDAM_SYSTEM_USER_PASSWORD: "Password12"
+      IDAM_URL: http://idam-api:5000
+      S2S_URL: http://service-auth-provider-api:8080
+      MANAGE_CASE_S2S_KEY: "${IDAM_KEY_AAC_MANAGE_CASE_ASSIGNMENT}"
+      MANAGE_CASE_S2S_AUTHORISED_SERVICES: ccd_gw,xui_webapp,ccd_data,fpl_case_service
+      CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api:4452
+      CCD_DATA_STORE_ALLOWED_SERVICE: ccd_gw
+      CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api:4451
+      MCA_DS_PROXY_URLS_ALLOWED_LIST: /searchCases.*,/internal/searchCases.*,/internal/cases.*
+      MCA_DEF_STORE_PROXY_URLS_ALLOWED_LIST: /api/display/challenge-questions.*
+      MCA_CONDITIONAL_APIS_CASE_ASSIGNMENTS_ENABLED: "true"
+      PRD_HOST: http://wiremock:8080
+      NOTIFY_MCA_API_KEY: "notify-test-api-key"
+      JAVA_TOOL_OPTIONS: '-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005'
```

Comment:
can be commented out to not start it always in debug mode, would be nice to use different debugger port that for ccd-data-store, in case dev want to debug both at the same time

Reply:
ahh forgot to commit the change to 5006